selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/languageserver/PhpFileEditingTest.java

```diff
@@ -79,11 +79,8 @@ public void setup() throws Exception {
 
   @Test
   public void checkMainFeaturesPhpLS() {
-    String intitPhpLanguageServerMessage =
-        String.format(
-            "Finished language servers initialization, file path '/%s'", PATH_TO_INDEX_PHP);
-
-    consoles.waitExpectedTextIntoConsole(intitPhpLanguageServerMessage);
+    consoles.waitExpectedTextIntoConsole(
```

Comment:
It's odd to use a constant in the other tests, but not here. If you're fixing this line anyway, why not make it the same as the other tests?